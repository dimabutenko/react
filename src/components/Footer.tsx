import React from "react";
import "./Footer.module.scss";

function Footer() {
    return (
        <footer>
            <p>Drag and drop to reorder list</p>
        </footer>
    );
}

export default Footer;