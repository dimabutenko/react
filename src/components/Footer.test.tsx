import React from 'react'
import { render } from '@testing-library/react'
import Footer from './Footer'

describe('Footer conpoonent tests', () => {
  test('Should render correctly', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
})
