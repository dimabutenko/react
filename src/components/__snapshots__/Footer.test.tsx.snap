// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer conpoonent tests Should render correctly 1`] = `
<div>
  <footer>
    <p>
      Drag and drop to reorder list
    </p>
  </footer>
</div>
`;
